### ISSUE TITLE
Reverse order

### ISSUE THREAD

**Issue by raousoft** [daveismyname/pagination#7]
In the actual script it does for example

previous 1 2 3 4 5 6 7 8 9 next

I want to get

next 9 8 7 6 5 4 3 2 1 previous

Pls

**Comment by jackttcoms** [comment]
As it is not done with an array you cannot do array_reverse and it is hard coded operators so if you want it to be the opposite i guess you should try flipping the < > around to get your desired result.

**Comment by raousoft** [comment]
> As it is not done with an array you cannot do array_reverse and it is hard coded operators so if you want it to be the opposite i guess you should try flipping the < > around to get your desired result.

Do you mean by < > ( next previous ?) I did it before it works , but the numbers still same from left to right.

**Comment by jackttcoms** [comment]
Btw, this was just a theory... i wouldn't say logic went into it exactly but i always say if something is coded one way and u want the other just do the opposite but of course you need to understand the code otherwise succeeding is well... tougher. Also why don't you just find a css compromise where you can flip it almost like bootstrap grid where you can do the order?

A while back I made a few adjustments to the class by adding some custom css addition so i can place the pagination and then go align----left......align----center.....align----right etc or whatever (basically any additional css u wanted to add and would love for @daveismyname to add it permanently as i think it would be very helpful to have as devs so u can easily add bootstrap code or just anything etc.

Thanks (kind of went off topic on the last bit but want @daveismyname to read it and see what he thinks)

💯

**Comment by raousoft** [comment]
I already tried all Css things, 
I understood the code somehow , I was able to flip some numbers (when pages are a lot )

I also added a redirect to home page , when you change (in url) the page that exceed the last page

but when the value is negative , let's say ?p=-1 , it will output:

`SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-10,5' at line 1
`

**Comment by jackttcoms** [comment]
why would u do -1 ? Personally tho, i will simply say this class is supposed to be simple and it is not built for your purpose i don't believe but maybe @daveismyname will reply and let u know his verdict.

**Comment by daveismyname** [comment]
I think CSS should do the job here, @jackttcoms I'd be interested in seeing your CSS.

Done a quick test using only css and seems to do the job well.

**Comment by jackttcoms** [comment]
my css? I'm not interested in the rtl that  @raousoft was after.

I was saying a nice addition to the class just to make it that littl bit more powerful would be to add another option (optional) that looks a bit like this:

`$pages = new Paginator('8', 'p', 'justify-content-center mb-0');` then you can reposition it as you please for the specific page it needed on etc. Not at all difficult but a nice addtion. What do you think @daveismyname

**Comment by daveismyname** [comment]
looks good to me, the only change I'd make is having $position as optional

```
public function __construct($perPage,$instance,$position = '')
```

**Comment by daveismyname** [comment]
@raousoft this seems to be all you need for rtl:

```
.pagination a, .pagination span.current, .pagination span.disabled {
    float: right !important;
}
```

**Comment by jackttcoms** [comment]
@daveismyname ye my bad but ye i think that would be a very positive update to your class in a future update. Give ur users more freedom etc.

**Comment by daveismyname** [comment]
yeah if you open a pull request I'll accept it so you get the credit for it.

**Comment by raousoft** [comment]
Solved Thank you so much Sir @daveismyname.
